app/components/ContactInfo.tsx

```diff
@@ -9,15 +9,15 @@ import "tailwindcss/tailwind.css";
 
 export default function ContactInfo() {
   return (
-    <div className="w-full flex flex-col items-center rounded shadow-md">
+    <div className="w-full flex flex-col items-center bg-opacity-70">
```

Comment:
I would honestly consider an even lower opacity than 70. The contact form is just completely dominated by the bg image right now. Maybe something like:
```suggestion
    <div className="w-full flex flex-col items-center bg-opacity-25">
```